Week6/app/src/main/java/com/oliverperez/blockbusterchallenge/app/Injection.kt

```diff
@@ -2,7 +2,8 @@ package com.oliverperez.blockbusterchallenge.app
 
 import com.oliverperez.blockbusterchallenge.model.MovieRepository
 import com.oliverperez.blockbusterchallenge.model.RoomRepository
+import kotlinx.coroutines.CoroutineScope
 
 object Injection {
-    fun provideMovieRepository(): MovieRepository = RoomRepository()
+    fun provideMovieRepository(scope: CoroutineScope): MovieRepository = RoomRepository(scope)
```

Comment:
You don't want to pass coroutine scope as a parameter. Coroutine scope is tied to a certain lifecycle, and when you pass it to another class that lives in another file, it's not clear what it refers to.